### PR TITLE
Documentation sys config

### PR DESCRIPTION
**Pull request by Neha-Sinha2305**
- Added a new file : docs/SystemConfiguration.md
- Modified README.md

### CODE REVIEW

File: docs/SystemConfiguration.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+# Test Configuration and Execution
+The tests are run using [Github Actions](https://github.com/features/actions) feature which makes it possible to automate, customize, and execute the software development workflows right in the repository. We utilize the **matrix testing strategy** to cover all the possible and logical combinations of the different properties and values that the components support.The [.env_override](../ansible/template/.env_override) file is used by the tests to override the values that are available in the .env_default file.<br/>
+
+As of today, the following tests are executed whenever a Pull request is created :<br/>
+
+1. test (consul_acl_enabled, consul_acl_allow, nomad_acl_enabled) : Here the tests are run on a system which is configured such that the consul policy is **enabled**, consul default acl policy is set to **allow** and nomad acl policy is **enabled**.<br/>
+
+2. test (consul_acl_enabled, consul_acl_allow, nomad_acl_disabled) : Here the tests are run on a system which is configured such that the consul policy is **enabled**, consul default acl policy is set to **allow** and nomad acl policy is **disabled**.<br/>
```
Comment (fredrikhgrelland) [inline docs/SystemConfiguration.md:33]:
```suggestion
2. test (consul_acl_enabled, consul_acl_allow, nomad_acl_disabled) : Here the tests are run on a system which is configured such that the consul policy is **enabled**, consul default acl policy is set to **allow** and nomad acl policy is **disabled**.


```

Comment (fredrikhgrelland) [reply docs/SystemConfiguration.md:33]: Break a line by using two newlines.

File: README.md [hunk 2]
```diff
@@ -47,6 +47,7 @@ The rest of the prerequisites are system-dependent:
 `make build` will build a vagrant box based on hashicorp/bionic64. The packaged box will be locally available at ´packer/output-hashistack/package.box´
 
 `make test` run tests by starting the [countdash](https://www.nomadproject.io/docs/integrations/consul-connect/) consul-connect example. If ´packer/output-hashistack/package.box´ does not exist, it will run ´make build´
+- Note: You can refer [SystemConfigurationDoc](docs/SystemConfiguration.md) inorder to get a comprehensive overview of the default configurations with which the sytstem is set up. The document also provides information about test configuration and steps for overriding the default system configuration.
```
Comment (pdmthorsrud) [inline README.md:50]:
```suggestion
- Note: You can refer to [SystemConfigurationDoc](docs/SystemConfiguration.md) in order to get a comprehensive overview of the default configurations with which the system is set up. The document also provides information about test configuration and steps for overriding the default system configuration.
```

File: docs/SystemConfiguration.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+#### **Nomad policies** added can be looked up in the following location: [nomad policies](../ansible/templates/nomad-policies/)
+
+#### **Consul policies** added can be looked up in the following location: [consul policies](../ansible/templates/consul-policies/)
+
+#### By default Consul Acl policies are enabled on the system and the default policy is set to allow.
+
+# How to override the default System Configuration
+
+When the vagrant box is provisioned it reads the data from the following environment file [.env_default](../ansible/template/.env_default) inorder to set
```
Comment (pdmthorsrud) [inline docs/SystemConfiguration.md:20]:
```suggestion
When the vagrant box is provisioned it reads the data from the following environment file [.env_default](../ansible/template/.env_default) in order to set

```

File: docs/SystemConfiguration.md (hunk 1 above)
Comment (pdmthorsrud) [inline docs/SystemConfiguration.md:33]: Could this be put in [README](README.md) instead, don't feel like `SystemConfiguration` is the right place to put it.

Comment (Neha-Sinha2305) [reply docs/SystemConfiguration.md:33]: Yeah..adding test section  just before the **Contribute** section

Comment (Neha-Sinha2305) [reply docs/SystemConfiguration.md]: Have incorporated the mentioned changes :+1:

File: README.md (hunk 2 above)
Comment (Neha-Sinha2305) [reply README.md]: I have fixed the typos, thanks :+1:

**Pull request merged**